src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,47 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <web-app xmlns="http://java.sun.com/xml/ns/javaee"
-	 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	 xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-	 version="3.0" metadata-complete="false">
-    
-    <display-name>Fedora-on-ModeShape</display-name>
-    
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+    version="3.0">
+
+    <display-name>Fedora Repository</display-name>
+
     <context-param>
         <param-name>contextConfigLocation</param-name>
-        <param-value>WEB-INF/classes/spring/auth-master.xml</param-value>
+        <param-value>WEB-INF/classes/spring/repository.xml</param-value>
     </context-param>
 
     <listener>
```

Comment:
Change to: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-webapp/src/main/webapp/WEB-INF/web.xml#L15-L17